docs/topics_5/web-create-project.adoc

```diff
@@ -0,0 +1,153 @@
+// Module included in the following assemblies:
+// * docs/web-console-guide_5/master.adoc
+[id='add_project_{context}']
+= Creating a project
+
+You create a project using a wizard-like interface.
+
+.Procedure
+
+. In the navigation pane, click *Projects*.
+. On the *Projects* page, click *Create project*.
++
+.Create project
+image::web-add-project_51.png[Create project]
+
+. Enter a unique name for the project.
+. Optional: Enter a description.
+. Click *Next*.
++
+.Add applications
+image::web-add-apps_51.png[Add applications]
++
+You can upload applications or register the server path that contains an application.
+
+. To upload applications:
+
+ .. In the *Upload* tab, click *Browse*.
+ .. Select the application files you want to upload.
+ .. Click *Close*.
+
+. To register a server path:
+
```

Comment:
If you want to register a server path, click the path, enter the "LABELS???? path of the ... and click Next.

I don't think you need that exploded Java line. It's obvious from the context what it means.